docs/data_providers/data_format/other_formats.md

```diff
@@ -95,7 +95,8 @@ If this is the first version of a dataset to be published, an administrator will
 do the following:
 
 * When the Excel metadata file passes checks, log in to Zenodo using our Zenodo curation
-  account (`the_safe_project`, tied to info@safeproject.net).
+  account (`the_safe_project`, tied to
```

Comment:
Once this is merged into the release branch, remind me to update the docs update branch, so that we handle possible conflicts with rewriting the docs across branches